README.md

```diff
@@ -62,3 +62,59 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
 - [React](https://reactjs.org/docs/getting-started.html)
 - [React Router](https://reacttraining.com/react-router/web)
 - [Bootstrap](https://getbootstrap.com/docs/4.4/getting-started/introduction/)
+
+
+## Deploy website to firebase
```

Comment:
I don't think we need all of these instructions.

We can keep the "Deploy to both test and production" part though

Reply:
So should I only keep the firebase deploy (test, production and both) comments and delete everything else?

I'm a bit confused. The commands for logging in and out of firebase is important, especially if you were working on another firebase project on the same system..

Reply:
Ok I added the script commands there instead, please take a look. Only the logout/login comments (needed in case someone is logged into another firebase project) and then the script commands (npm run <script>).

Reply:
Made it shorter.  Take a look

Reply:
Yep, I have looked at it. Seems good!